package.json

```diff
@@ -8,7 +8,7 @@
     "dist/client"
   ],
   "scripts": {
-    "build": "svc-tools build --mode production",
+    "build": "npm run clean && svc-tools build --mode production",
```

Comment:
Can we revert this change. If this is something we want to do, it should be part of the tooling.

Have been trying to avoid `&&`'s on the "core"/common scripts.

Reply:
For local testing, `npm run build` doesn't remove the previous `dist` build.